app/src/main/java/org/jellyfin/androidtv/ui/browsing/StdGridFragment.java

```diff
@@ -514,8 +519,35 @@ public void dismiss() {
         }
     }
 
-    protected void setupEventListeners() {
+    protected BaseGridView.OnKeyInterceptListener mKeyListener = new BaseGridView.OnKeyInterceptListener() {
+
+        @Override
+        public boolean onInterceptKeyEvent(KeyEvent event) {
+            if (mCurrentItem == null || event.getAction() != KeyEvent.ACTION_UP) {
+                return false;
+            }
+            if ((event.getKeyCode() == KeyEvent.KEYCODE_MEDIA_PLAY || event.getKeyCode() == KeyEvent.KEYCODE_MEDIA_PLAY_PAUSE) && BaseItemUtils.canPlay(mCurrentItem.getBaseItem())) {
+                mediaManager.getValue().setCurrentMediaAdapter(mGridAdapter);
+                mediaManager.getValue().setCurrentMediaPosition(mCurrentItem.getIndex());
+                mediaManager.getValue().setCurrentMediaTitle(mFolder.getName());
+                //default play action
+                mHandler.postDelayed(() -> PlaybackHelper.playOrPlayNextUp(mCurrentItem.getBaseItem(), requireActivity()), 10);
+                return true;
+            }
```

Comment:
The keyprocessor already deals with KEYCODE_MEDIA_PLAY, why did you make a separate implementation here?

Reply:
I was not sure if changing from `createPlayMenu`  to a actual play action was just a oversight in the org. code or had some use in other views?

My goal was to get a actual play action, not just a menu if u hit the play-button, since there is a dedicated menu action otherwise.

I can rework this in the keyprocessor for series/seasons/episode/movie/video cases?

Reply:
ah also the org. code for BaseActivity did set those parameters, so i just mimic-ed this logic.
```
                mediaManager.getValue().setCurrentMediaAdapter(mGridAdapter);
                mediaManager.getValue().setCurrentMediaPosition(mCurrentItem.getIndex());
                mediaManager.getValue().setCurrentMediaTitle(mFolder.getName());
```

Reply:
> My goal was to get a actual play action, not just a menu if u hit the play-button, since there is a dedicated menu action otherwise.

It only shows a menu when there's multiple options, I always use the play button on my remote to immediately start items from the play next home section. So I don't see the need for duplicate code here.

Reply:
Not sure what you mean?
Ofc it plays from the play next home section, since it only actually shows possible play-next items?

So whats the conclusion now? Do i move the new play-code to the respective basetypes inside the keyhandler?

Reply:
Ok just noticed that the "play" case for any collection basetype makes no sense, it always just brings-up up a menu, even for music.

That's not how the webclient or any music-player works, if i hit "play" on a album/playlist i expect that it actually starts. 
So we should change all those collection cases to a actual play, so this is consistent with the webclient and normal user expectations.